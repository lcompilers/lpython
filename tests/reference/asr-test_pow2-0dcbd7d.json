{
    "basename": "asr-test_pow2-0dcbd7d",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_pow2.py",
    "infile_hash": "0df16d4f512305265cb2924e18db5753a4c422da3a7624f74f71d26f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_pow2-0dcbd7d.stderr",
    "stderr_hash": "1d1d3a4a308e068439ac40685fcad6f29750560722b910be3341ce3c",
    "returncode": 2
}