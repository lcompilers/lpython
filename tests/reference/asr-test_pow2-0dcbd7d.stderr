style suggestion: Could have used '**' instead of 'pow'
  --> tests/errors/test_pow2.py:12:11
   |
12 |     print(pow(a, b, c, d))
   |           ^^^^^^^^^^^^^^^ '**' could be used instead

semantic error: Arguments do not match for any generic procedure, pow
  --> tests/errors/test_pow2.py:12:11
   |
12 |     print(pow(a, b, c, d))
   |           ^^^^^^^^^^^^^^^ 
